superset/viz.py

```diff
@@ -2041,6 +2044,9 @@ class BaseDeckGLViz(BaseViz):
     credits = '<a href="https://uber.github.io/deck.gl/">deck.gl</a>'
     spatial_control_keys = []
 
+    def handle_nulls(self, df):
+        pass
```

Comment:
let's raise a `raise NotImplementedError()`

Reply:
oh here I actually want to do nothing